### ISSUE TITLE
Lock window size option

### ISSUE THREAD

**Issue by flukejones** [sixtyfpsui/sixtyfps#227]
I saw through a general repo search there is `SIXTYFPS_FULLSCREEN` for exclusive fullscreen mode (awesome!). But is there a similar option to lock the window size completely? This might be handy for instances where an app spawns a new window as a popup.

**Comment by ogoffart** [comment]
This is currently possible with a layout that has minimum and maximum size.
E.g:
```
X := Window {
  VerticalLayout {
    Rectangle {
        width: 300px;
        height: 350px;
    }
  }
}
```

This locks the size of the window.

Ideally it should also work by setting minimum-width / maximum-width and co. on the windows, but the code that allows it is currently commented out.

https://github.com/sixtyfpsui/sixtyfps/blob/f7cb67d086bf5714b75241bde78833c207578e7a/sixtyfps_compiler/passes/lower_layout.rs#L41-L43

That's because the `height` and `width` property means that the size is fixed on other element. But in the Window we meant that to be the initial size. And that's why minimum-* and maximum-* are also disabled because that code would otherwise fix the size also for just height and width.
I'm tempted to simply enable this code again, and the initial size should be set with preferred-width and preferred-height again.
@tronical : what do you think?

**Comment by tronical** [comment]
I agree to re-enable this code again

**Comment by ogoffart** [comment]
Closed by commit fbdbe69

Thanks for the bug report.
